float64 micHeight
float64 micOrientation